### ISSUE TITLE
Perpetual GBM model

### ISSUE THREAD

**Issue by dean-sh** [Nixtla/mlforecast#461]
### Description

PerpetualBooster is a new gradient boosting machine (GBM) algorithm which doesn't need hyperparameter optimization unlike other GBM algorithms.
It includes a `budget` parameter which can be tweaked to optimise the search.

From their results, it seems to be faster for training/inference against a tuned LightGBM with similar performance, and mostly better than AutoGluon across 10 datasets.

https://github.com/perpetual-ml/perpetual

### Use case

I'd be keen to test this as a drop-in replacement to any other GBM model and integrate it to nixtla MLforecast natively.

**Comment by jmoralez** [comment]
Hey. If it implements `fit` and `predict` you can just provide it directly to the `models` argument.

**Comment by dean-sh** [comment]
Would it support quantiles (confidence intervals)?
